services/oc-guytonmodel/repocreator.sh

```diff
@@ -0,0 +1,40 @@
+CURRENTDIR=${pwd}
+
+# step 1: name of the remote repo. Enter a SINGLE WORD ..or...separate with hyphens
```

Comment:
looks cool, we should add that file in the cookie cutter.
could you create a PR in there as well?